Vagrantfile

```diff
@@ -0,0 +1,38 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+@script = <<SCRIPT
+apt-get update
+apt-get install -y apache2 git curl php5-cli php5 libapache2-mod-php5
+echo "
+<VirtualHost *:80>
+    ServerName zf2-tutorial.localhost
+    DocumentRoot /var/www/zf/public
+    <Directory /var/www/zf/public>
+        DirectoryIndex index.php
+        AllowOverride All
+        Order allow,deny
+        Allow from all
+    </Directory>
+</VirtualHost>
+" > /etc/apache2/sites-available/000-default.conf
+a2enmod rewrite
+service apache2 restart
+cd /var/www/zf
+curl -Ss https://getcomposer.org/installer | php
+php composer.phar install --no-progress
+SCRIPT
```

Comment:
Could we print an echo line with something like "Visit http://localhost:8080 in your browser for to view the application"

I feel the detail of to use port `8080` is not advised